SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -216,7 +216,7 @@ Quarks {
 
 	*link { |path|
 		path = path.withoutTrailingSlash;
-		if(LanguageConfig.includePaths.includesEqual(path).not, {
+		if(LanguageConfig.includePathsAbsolute.includesEqual(path).not, {
```

Comment:
if you feel like it, feel free change to a uniform "if" style, to fit the rest of your changes:

`if(LanguageConfig.includePathsAbsolute.includesEqual(path).not) {`

it would be good to have a uniform style at least in one file.